docs/headers/fmi3LsBusCan.h

```diff
@@ -0,0 +1,375 @@
+
+/**
+ * FMI data type to be used for the CAN status kind.
+ */
+typedef fmi3UInt8 fmi3LsBusCanStatusKind;
+
+#define FMI3_LS_BUS_CAN_STATUS_PARAM_STATUS_KIND_ERROR_ACTIVE ((fmi3LsBusCanStatusKind)0x1)
+
+#define FMI3_LS_BUS_CAN_STATUS_PARAM_STATUS_KIND_ERROR_PASSIVE ((fmi3LsBusCanStatusKind)0x2)
+
+#define FMI3_LS_BUS_CAN_STATUS_PARAM_STATUS_KIND_BUS_OFF ((fmi3LsBusCanStatusKind)0x3)
+
+/**
+ * Can status operation.
+ */
+typedef struct
+{
+    fmi3LsBusOperationHeader header; /**< Message header */
+    fmi3LsBusCanId id;               /**<  CAN message ID. */
+    fmi3LsBusCanStatusKind status;   /**<  Status Kind. */
+} fmi3LsBusCanOperationStatus;
+
+static_assert(sizeof(fmi3LsBusCanOperationStatus) == (5 + 4 + 1),
+              "'fmi3LsBusCanOperationStatus' does not match the expected data size");
+
+/**
+ * FMI data type to be used for baudrate settings.
+ */
+typedef fmi3UInt32 fmi3LsBusCanBaudrate;
+
+/**
+ * FMI data type to be used for the configuration parameter type.
+ */
+typedef fmi3UInt8 fmi3LsBusCanConfigParameterType;
+
+/**
+ * Definitions of configuration parameter type.
+ */
+#define FMI3_LS_BUS_CAN_CONFIG_PARAM_TYPE_CAN_BAUDRATE ((fmi3LsBusCanConfigParameterType)0x1)
+#define FMI3_LS_BUS_CAN_CONFIG_PARAM_TYPE_CANFD_BAUDRATE ((fmi3LsBusCanConfigParameterType)0x2)
+#define FMI3_LS_BUS_CAN_CONFIG_PARAM_TYPE_CANXL_BAUDRATE ((fmi3LsBusCanConfigParameterType)0x3)
+#define FMI3_LS_BUS_CAN_CONFIG_PARAM_TYPE_ARBITRATION_LOST_BEHAVIOR ((fmi3LsBusCanConfigParameterType)0x4)
+
+/**
+ * FMI data type to be used for the CAN configuration options.
+ */
+typedef fmi3UInt8 fmi3LsBusCanArbitrationLostBehavior;
+
+#define FMI3_LS_BUS_CAN_CONFIG_PARAM_ARBITRATION_LOST_BEHAVIOR_BUFFER_AND_RETRANSMIT \
+    ((fmi3LsBusCanArbitrationLostBehavior)0x1)
+
+#define FMI3_LS_BUS_CAN_CONFIG_PARAM_ARBITRATION_LOST_BEHAVIOR_DISCARD_AND_NOTIFY \
+    ((fmi3LsBusCanArbitrationLostBehavior)0x2)
+
+/**
+ * Can configuration operation.
+ */
+typedef struct
+{
+    fmi3LsBusOperationHeader header; /**< Message header */
+    fmi3LsBusCanConfigParameterType parameterType; /**< Defines the current configuration parameter.
+                                                        Note: Only one parameter can be set per Configuration operation. */
+    union
+    {
+        fmi3LsBusCanBaudrate baudrate;
+        fmi3LsBusCanArbitrationLostBehavior arbitrationLostBehavior;
+    };
+} fmi3LsBusOperationCanConfiguration;
+
+static_assert(sizeof(fmi3LsBusOperationCanConfiguration) >= (5 + 4 + 1),
+              "'fmi3LsBusOperationCanConfiguration' does not match the expected data size");
+
+/**
+ * Can wakeup operation.
+ */
+typedef struct
+{
+    fmi3LsBusOperationHeader header; /**< Message header */
+} fmi3LsBusCanOperationWakeup;
+
+static_assert(sizeof(fmi3LsBusCanOperationWakeup) == (5),
+              "'fmi3LsBusCanOperationWakeup' does not match the expected data size");
+
+#pragma pack()
+
+#ifdef __cplusplus
```

Comment:
There is no opening bracket. As no C++ specifics like extern "C" are required this can be removed